MessengerOnFileForLinux/UserService/test/SignInTest.cpp

```diff
@@ -1,4 +1,4 @@
-/*#include <gtest/gtest.h>
+#include <gtest/gtest.h>
 
```

Comment:
Imho jest za mało testów, to że przechodzimy przez każde miejsce nie oznacza, że jest wystarczająco pokryte. Na pierwszy rzut oka imho brakuje:
- rejestracja gdzie pierwsze podane hasło jest inne od drugiego
- rejestracja i logowanie pustym hasłem

Dodatkowo pierwsze dwie linijki z każdego testu do funkcji, która jest potem wołana z parametrem.

Reply:
Z tymi linijkami już próbowałem przerzucać, ale niestety coś wtedy to nie chce działać. Sprawdzę to jeszcze raz dla pewności.

Testów bym nie dopisywał. Po refaktorze klasa będzie działała ciut inaczej i teraz napisane testy i tak byłyby do sporego aligmentu.

Reply:
Co do pierwszej kropki to nawet jest taki test, tylko że on pierwsze wprowadza zle hasla, a za drugą próbą już poprawne. Tak to jest zrobione bo taka jest aktualnie implementacja. Po refaktorze chcialbym żeby takie coś po prostu po pierwszej próbie zwróciło nieudaną rejestrację, i klasa nadrzędna sterujaca rejestracją i logowanie ponowi próbę rejestracji.

Reply:
Rejestracja pustym hasłem czy logowanie pustym hasłem nie przejdzie, bo ncurse tego nie puszcza. Więc nawet nie można czegoś takiego przetestować, bo test utknie w programie w lini pobierającej hasło.

Reply:
zrób taska na to by rozkminić wszystkie kombinacje

Reply:
ok